### DISCUSSION TITLE
Wallet Problem

### DISCUSSION THREAD

**Discussion by yeasin850** [blockscout/blockscout#9755] [Support]
Sir Assalamualaikum.
Sir, there is a wallet problem in my Sidra account. There were two ways to add a wallet, one was Sidra's own Sidra wallet and the other was Metamask wallet. I added two wallets. The problem is that the Sidra wallet is not showing Sidra tokens, but the wallet I added in Metamask is showing all tokens. In this situation, if you do not help now, then my pain and dreams will be completely in vain. I seek your help from the bottom of my heart. Plz Sir Help.
![Screenshot_20240330-055031](https://github.com/blockscout/blockscout/assets/72614776/473f46e0-d2af-42b9-82d3-48448f6b9e92)
![Screenshot_20240330-055037](https://github.com/blockscout/blockscout/assets/72614776/8042c623-b0db-4044-9960-dd8210740c70)

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.
